src/test/suite/fixtures/recorded/scopes/token/clearTokenBackward.yml

```diff
@@ -0,0 +1,26 @@
+languageId: plaintext
+command:
+  spokenForm: clear token backward
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: relativeScope
+          scopeType: {type: token}
+          offset: 0
+          length: 1
+          direction: backward
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: aaa.
```

Comment:
This test should probably be done with the tokens in reverse order to make sure we are ignoring preference?